### PR TITLE
docs: Edit README / LICENSE

### PR DESCRIPTION
**Pull request by jvirgovic**
## Description
I think that the README section deserved a little bit of simplifying, as it was unnecessary long. (Check: Average reading human attention span)

## Checklist:
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

- added missing information to LICENSE
- removed one video of executing Web Interface, as I don't think we need 2 of them since they were showing exactly the same thing.
- removed long section of text at the beginning. Now we get straight to the point of what grapheneX is for. (Most people who'll use this project already know what hardening is, or can easily google that up)
- removed screenshots, because we already have a lot of video demonstrations how grapheneX works, thus the potential end user can already see how the CLI / Web UI looks.

<b>Please feel free to disagree with me on this one - Maybe I just prefer more minimalist approach.</b>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -341,18 +333,6 @@ docker build -f docker/Dockerfile -t graphenex .
...
-![Screenshot I](https://user-images.githubusercontent.com/24392180/60615641-47866e80-9dd8-11e9-92fe-dfa8874db8fd.jpg)
-
-![Screenshot II](https://user-images.githubusercontent.com/24392180/60601258-7e4d8c00-9dba-11e9-93e1-96411b76b462.png)
-
-![Screenshot III](https://user-images.githubusercontent.com/24392180/60625283-9be51a80-9de7-11e9-99ea-46875d8b3ed8.JPG)
-
-![Screenshot IV](https://user-images.githubusercontent.com/24392180/60625314-bae3ac80-9de7-11e9-98ef-e5fa69c3be38.JPG)
-
-![Screenshot V](https://user-images.githubusercontent.com/24392180/60625373-eb2b4b00-9de7-11e9-9e9a-ab1869af9b1a.JPG)
-
```
Comment (orhun) [inline README.md]: Let's add these back :) There is some nostalgic value attached to these.

File: README.md [hunk 2]
```diff
@@ -99,8 +93,6 @@ Execute the `grapheneX` with the `-w` or `--web` argument in order to start the
 
 • Web interface has the authentication system that requires an `access token`. Once the user verifies her/his identity with the given token at the shell, grapheneX creates a session for further use.
 
-![GrapheneX Web Interface I](https://user-images.githubusercontent.com/24392180/61898886-fd019900-af22-11e9-9d1e-e2a58bee6651.gif)
-
```
Comment (orhun) [inline README.md]: Let's add this back as well :)

File: LICENSE [hunk 3]
```diff
@@ -652,7 +652,7 @@ Also add information on how to contact you by electronic and paper mail.
   If the program does terminal interaction, make it output a short
 notice like this when it starts in an interactive mode:
 
-    <program>  Copyright (C) <year>  <name of author>
+    grapheneX  Copyright (C) 2023  orhun
```
Comment (orhun) [inline LICENSE]:
```suggestion
    grapheneX  Copyright (C) 2019-2023 Orhun Parmaksız
```

File: LICENSE [hunk 4]
```diff
@@ -631,8 +631,8 @@ to attach them to the start of each source file to most effectively
 state the exclusion of warranty; and each file should have at least
 the "copyright" line and a pointer to where the full notice is found.
 
-    <one line to give the program's name and a brief idea of what it does.>
-    Copyright (C) <year>  <name of author>
+    grapheneX - Automated System Hardening Framework
+    Copyright (C) 2023  orhun
```
Comment (orhun) [inline LICENSE]:
```suggestion
    Copyright (C) 2019-2023 Orhun Parmaksız
```

File: README.md (hunk 1 above)
Comment (jvirgovic) [reply README.md]: Sure.

**Review by orhun — APPROVED** [review]
Thanks!

**Pull request merged**